### ISSUE TITLE
Lack of Signature

### ISSUE THREAD

**Issue by mkschnitzel** [ReactiveMarkets/NLog.Targets.ElasticSearch#67]
Hello,

in our project we want to add this library to the GAC.
Because libraries need to be signed to be added to the GAC, we are not able to add your library to the GAC.
Would it be possible to sign the library?

Kind regards

**Comment by markmcdowell** [comment]
I don't think this is something that we'd do, I'd discourage use of the gac anyway.

**Comment by realhawker** [comment]
I think threre are situations other than the GAC where you need a strong name for the assembly e.g. when you are writting an extension for the Windows Explorer like a ShellOverlayIconHandler. Adding a strong name to the assembly is easy and not having it limits the usage of the libray in an unnecessary way.
